src/generated/resources/data/forge/tags/items/ores/coal.json

```diff
@@ -1,6 +1,7 @@
 {
   "replace": false,
   "values": [
-    "minecraft:coal_ore"
+    "minecraft:coal_ore",
```

Comment:
Shouldn't all the `ores/x` tags use the `#minecraft:x_ores` tags in them, like `#forge:ores/gold` does?

Reply:
Yes, you are correct. I mixed up the 1.16 and 1.17 folders, so I didn't even realize a bunch of these new block tags existed 🤦‍♂️